plugins/Feedback/API.php

```diff
@@ -60,6 +62,73 @@ public function sendFeedbackForFeature($featureName, $like, $message = false)
         $this->sendMail($subject, $body);
     }
 
+    /**
+     * Sends feedback for a specific feature to the Matomo team or alternatively to the email address configured in the
+     * config: "feedback_email_address".
+     *
+     * @param $question
+     * @param string|bool $message A message containing the actual feedback
+     * @throws \Piwik\NoAccessException
+     * @throws \Exception
+     */
+    public function sendFeedbackForSurvey($question,  $message = false)
+    {
+        Piwik::checkUserIsNotAnonymous();
+        Piwik::checkUserHasSomeViewAccess();
+
+        // if message is test content then don't send email
+        if ($message == 'test') {
+            $feedbackReminder = new FeedbackReminder();
+            $feedbackReminder->setUserOption("-1");
+            return 'success';
+        }
+
+        if ($message == '' || strlen($message) < 10) {
+            return Piwik::translate("Feedback_MessageBodyValidationError");
+        }
+
+
+        $featureName = $this->getEnglishTranslationForFeatureName($question);
+
+
+        $body = sprintf("Question: %s\n", $featureName);
+
+        $feedbackMessage = "";
+        if (!empty($message) && $message != 'undefined') {
+            $feedbackMessage = sprintf("Answer:\n%s\n", trim($message));
+        }
+        $body .= $feedbackMessage ? $feedbackMessage : " \n";
+
```

Comment:
just some style improvements

```suggestion

        $featureName = $this->getEnglishTranslationForFeatureName($question);
        $body = sprintf("Question: %s\n", $featureName);
        $feedbackMessage = "";
        
        if (!empty($message) && $message !== 'undefined') {
            $feedbackMessage = sprintf("Answer:\n%s\n", trim($message));
        }

        $body .= $feedbackMessage ? $feedbackMessage : " \n";

```